src/main/java/org/spongepowered/common/interfaces/IMixinWorld.java

```diff
@@ -100,4 +103,12 @@
 
     void notifyBlockOfStateChange(BlockPos notifyPos, Block sourceBlock, BlockPos sourcePos);
 
+    @Nullable
+    EntityPlayer getClosestPlayerToEntityWhoAffectsSpawning(net.minecraft.entity.Entity entity, double distance);
+
+    @Nullable
+    EntityPlayer getClosestPlayerWhoAffectsSpawning(double x, double y, double z, double distance);
+
+    boolean isAnyPlayerWithinRangeAtWhoAffectsSpawning(double x, double y, double z, double range);
```

Comment:
If you want to stick as closely to the original name of that method then you could but the parts in proper order. 
`isAnyPlayerWhoAffectsSpawningWithinRangeAt`

but @kenzierocks will work as well. (I'm not a grammar expert and can't tell which version is better, but the current naming is just strage)